internal/dataplane/translator/subtranslator/httproute.go

```diff
@@ -321,6 +329,7 @@ func getHTTPRouteHostnamesAsSliceOfStringPointers(httproute *gatewayapi.HTTPRout
 // to list of kongstate.Route.
 func translateHTTPRouteRulesMetaToKongstateRoutes(
 	rulesMeta []httpRouteRuleMeta,
+	supportRedirectPlugin bool,
```

Comment:
Passing options struct here would also improve readability of call sites.

Reply:
done.